src/core/Akka.Tests/Util/ByteStringSpec.cs

```diff
@@ -57,8 +60,10 @@ public void A_ByteString_must_be_sequential_when_slicing_from_start()
         [Fact]
         public void A_ByteString_must_be_sequential_when_slicing_from_index()
         {
-            Prop.ForAll((ByteString a, ByteString b) => (a + b).Slice(a.Count).SequenceEqual(b))
```

Comment:
Some of these property-based tests no longer pass because, as should be no surprise, the `ByteString.Slice` method will no longer work with arbitrary index and count values - hence why certain random-generated values fail during testing now.